src/sdk/clients/createNexusClient.ts

```diff
@@ -222,6 +224,22 @@ export async function createNexusClient(
     chain,
     key,
     name,
+    paymasterContext: paymasterContext
```

Comment:
Don't understand this bit. Why isn't this simply `paymasterContext`? Haven't you just broken support for all other paymasters in doing this?

Reply:
Basically if "paymasterContext" is not undefined here, we use the fields that are set inside of it and we add the ones that never change so that the user only have to provide the fee token address and some other optional fields like expiry. If there is not paymaster context it defaults to undefined and viem will default it for sponsored mode, it does not break any support for other paymasters. 

Anyway, thanks for pointing this out, I have changed it to a more elegant approach.